### DISCUSSION TITLE
Alternate mod key

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [i3/i3#5068] [Q&A] [answered]
## Problem

My laptop doesn't have Super_L, but my main kb I normally use has. So I want to keep
```
set $mod Mod4
```
But I also want to have an alternate version of it, for example, `Alt+Shift` or `Alt+Control`. This solution
```
bindsym Mod1+Control exec --no-startup-id xdotool key --clearmodifiers $mod
```
didn't work for me, because as far as I understand, `i3` doesn't catch `Mod1+Control` (but if I add, say, `+A`, it catches the whole combination).

I know it's possible to duplicate all bindings with another modifier, and it does work (tested with one). But can I do it without this big copy pasting?

Also, thanks for this great WM!

**Comment by Airblader** [comment, marked as answer]
If you only need one of the bindings to be active at a time, you can use a variable for the modifier key and write something like a udev script to reload the i3 config with the correct one.

If you want both versions to be bound at the same time, i3 has no built-in way of doing this (other than writing out all key bindings twice). Something like [i3bang](https://github.com/tckmn/i3bang) would make this easy (not sure of the state of that project).

**Reply by WhiteBlackGoose** [reply]
Alright, I see. Thanks
